src/Controller/Component/CsrfComponent.php

```diff
@@ -56,7 +56,7 @@ class CsrfComponent extends Component
         'cookieName' => 'csrfToken',
         'expiry' => 0,
         'secure' => false,
-        'httpOnly' => false,
+        'httpOnly' => true,
```

Comment:
How would the attacker submit the form? The submit needs to be initiated by the user's browser (the one having the session information).

Again, let's imagine that the attacker is able to run any code from the cake page and send the information to their servers... What is the difference between being able to read the cookie or simply read the hidden value in the form?